mmv1/third_party/terraform/website/docs/d/parameter_manager_parameter_version_render.html.markdown

```diff
@@ -38,7 +38,7 @@ The following attributes are exported:
 
 * `parameter_data` - The Parameter data.
 
-* `render_parameter_data` - The Rendered Parameter Data specifies that if you use `__REF__()` to reference a secret and the format is JSON or YAML, the placeholder `__REF__()` will be replaced with the actual secret value. However, if the format is UNFORMATTED, it will stay the same as the original `parameter_data`.
+* `rendered_parameter_data` - The Rendered Parameter Data specifies that if you use `__REF__()` to reference a secret and the format is JSON or YAML, the placeholder `__REF__()` will be replaced with the actual secret value. However, if the format is UNFORMATTED, it will stay the same as the original `parameter_data`.
```

Comment:
can you share where you're getting this? It's not clear where this is coming from.